server/routes/viewingLicences/handlers/viewCases.ts

```diff
@@ -74,12 +76,11 @@ export default class ViewAndPrintCaseRoutes {
   GET_WITH_EXCLUSIONS = async (req: Request, res: Response): Promise<void> => {
     const { user } = res.locals
     const { caseloadsSelected = [] } = req.session
-    const view = req.query.view || 'prison'
     const allPrisons = await this.prisonerService.getPrisons()
     const activeCaseload = allPrisons.filter(p => p.agencyId === user.activeCaseload)
     const prisonCaseloadToDisplay = caseloadsSelected.length ? caseloadsSelected : [activeCaseload[0].agencyId]
 
-    const cases = await this.caseloadService.getOmuCaseload(user, prisonCaseloadToDisplay, view.toString())
+    const cases = await this.caseloadService.getOmuCaseload(user, prisonCaseloadToDisplay)
```

Comment:
this should probably still use view to determine which caselist, similar to above: 
```
    const cases = view === 'prison' ? caselist.getPrisonView() : caselist.getProbationView()
```

Reply:
so the exclusion list will respect the same query params as used on the main caselist view

Reply:
done